python/ray/ml/examples/xgboost_example.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+from typing import Tuple
+
+import pandas as pd
+
+import ray
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.predictors.integrations.xgboost import XGBoostPredictor
+from ray.ml.train.integrations.xgboost import XGBoostTrainer
+from ray.data.dataset import Dataset
+from ray.ml.result import Result
+from ray.ml.constants import TRAIN_DATASET_KEY
+from ray.ml.preprocessors import StandardScaler
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+
+
+def prepare_data() -> Tuple[Dataset, Dataset, Dataset]:
```

Comment:
seems to be duplicated from above. Maybe make this a common test util that already returns a `datasets`-arg ready dict?

Reply:
Will follow up on this